src/components/dictionaryConcepts/components/ExternalMapping.jsx

```diff
@@ -28,6 +28,7 @@ class ExternalMapping extends Component {
                 onChange={this.handleChange}
                 id="map_type"
                 required
+                disabled
```

Comment:
What is disabled doing here?

Reply:
disabled was put there to make the select option uneditable since the edit was removed already.

Reply:
Was the goal to just disable or completely remove?

Reply:
The goal was to remove completely